### PR TITLE
Clean apt archives and source directories in Dockerfile

### PR DESCRIPTION
**Pull request by nhlfr**
Signed-off-by: Michal Rostecki <michal@kinvolk.io>

### CODE REVIEW

**Comment by hqhq** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -34,7 +35,8 @@ ENV CRIU_VERSION 1.7
 RUN mkdir -p /usr/src/criu \
     && curl -sSL https://github.com/xemul/criu/archive/v${CRIU_VERSION}.tar.gz | tar -v -C /usr/src/criu/ -xz --strip-components=1 \
     && cd /usr/src/criu \
-    && make install-criu
+    && make install-criu \
+    && make clean
```
Comment (cyphar) [inline Dockerfile]: Silly question, why don't we just do an `rm -rf /usr/src/criu`? Once it's installed do we need the source code lying  around?

Comment (nhlfr) [inline Dockerfile]: Good idea, done.

**Comment by mrunalp** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**